Traceback (most recent call last):
  File "main.py", line 263, in <module>
    _, model_description['bohamiann']['model_path'] = 'src/stored_models/F_r_i_ _S_e_p_ _1_3_ _0_1_:_1_1_:_3_2_ _2_0_1_9_BOHAMIANN' #run_model(model_description['bohamiann']['config'], save_model_str=get('run_dir'), config_type='BOHAMIANN')
ValueError: too many values to unpack (expected 2)
